packages/google_sign_in/google_sign_in_ios/example/ios/Podfile

```diff
@@ -26,17 +26,14 @@ end
 require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)
 
 # Suppress warnings from transitive dependencies that cause analysis to fail.
-pod 'AppAuth', :inhibit_warnings => true
-pod 'GTMAppAuth', :inhibit_warnings => true
+inhibit_all_warnings!
```

Comment:
These lines inhibit warnings but also add `AppAuth` and `GTMAppAuth` as CocoaPod dependencies:

```
pod 'AppAuth', :inhibit_warnings => true
pod 'GTMAppAuth', :inhibit_warnings => true
```

If the SwiftPM feature is enabled, this caused duplicate modules and the build would fail.

This is a temporary workaround. In the future, we'll remove CocoaPod integration from these example apps. That's tracked by https://github.com/flutter/flutter/issues/148021.